csharp/sources/Valkey.Glide/GlideClient.cs

```diff
@@ -74,10 +74,13 @@ public async Task<string> Info(InfoOptions.Section[] sections)
         => await Command(Request.Info(sections));
 
     public async Task<ValkeyValue> EchoAsync(ValkeyValue message, CommandFlags flags = CommandFlags.None)
-    {
-        Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
-        return await Command(Request.Echo(message));
-    }
+        => await Command(Request.Echo(message));
+
+    public async Task<TimeSpan> PingAsync(CommandFlags flags = CommandFlags.None)
+        => await Command(Request.Ping(flags));
+
+    public async Task<TimeSpan> PingAsync(ValkeyValue message, CommandFlags flags = CommandFlags.None)
+        => await Command(Request.Ping(message, flags));
 
     public async Task<bool> KeyMoveAsync(ValkeyKey key, int database, CommandFlags flags = CommandFlags.None)
         => await Command(Request.KeyMoveAsync(key, database));
```

Comment:
Ideally need to align all request names, they are neither sync or async. It is just a struct.
```suggestion
        => await Command(Request.KeyMove(key, database));
```
But don't worry about it now, we can make another PR for this.